scripts/env-data.sh

```diff
@@ -90,6 +101,10 @@ if [ -z "${TOPOLOGY}" ]; then
 	TOPOLOGY=true
 fi
 # Replication settings
+
+if [ -z "${REPLICATION}" ]; then
+	REPLICATION=true
+fi
```

Comment:
I thought for backward compatibility we could let the default be true. If we change it to false how are we going to inform users who have replication running that when they upgrade they will have to change it to `true`. Maybe we can change it to false when we upgrade to a major PG release since people would have to go through the DB migration process